hangman/lib/hangman/game.ex

```diff
@@ -0,0 +1,75 @@
+defmodule State do
+    defstruct game_state: :initializing,
+     turns_left: 7,
+     letters: [],
+     used: [],
+     word: "",
+     last_guess: ""
+end
+
+defmodule Hangman.Game do
+  @moduledoc """
+  The Hangman.Game module provides an api to allow users to create a game of
+  hangman, make guesses, and see the progression off the game.
+  """
+
+  def new_game() do
```

Comment:
## +2

Thank you. You are one of the few people to have implemented `new_word/1`, and one of the even fewer to use it in the implementation of `new_word/0`